src/test/java/com/google/jenkins/plugins/computeengine/InstanceConfigurationTest.java

```diff
@@ -184,68 +188,63 @@ public void testConfigRoundtrip() throws Exception {
   public void testInstanceModel() throws Exception {
     Instance i = instanceConfiguration(MIN_CPU_PLATFORM).instance();
```

Comment:
If your changing it already please change `i` -> `instance`

Reply:
Done.